beacontools/backend/__init__.py

```diff
@@ -0,0 +1,8 @@
+import sys
+
+if sys.platform.startswith("linux"):
+    from .bluez import *
+elif sys.platform.startswith("freebsd"):
+    from .freebsd import *
+else:
+    raise NotImplementedError("Scanning not supported on this platform")
```

Comment:
With this you are effectively removing support for Windows and macOS which are also supported by the pybluez backend. Please change this so that only on FreeBSD that backend is used and bluez otherwise.
```suggestion
if sys.platform.startswith("freebsd"):
    from .freebsd import *
else:
    from .bluez import *
```

Reply:
No, they were never supported!! (i wrote this in the commit message)

The `bluetooth._bluetooth` module that this code imports is literally the [C extension that binds to bluez libs](https://github.com/karulis/pybluez/blob/06f3c83dcd7857c7f97aa1f882357725fc4342d4/setup.py#L80-L84).

PyBluez mentions Windows and Mac in the README because it has some generic Bluetooth Classic support on all three platforms. That's not what you're using.

Reply:
Ah ok, I never tried it and thus didn't realize that it doesn't work in these platforms.